lib/partition_setup.pm

```diff
@@ -123,6 +123,7 @@ sub addpart {
         send_key_until_needlematch "partition-selected-$args{fsid}-type", 'up';
     }
     if ($args{mount}) {
+        send_key 'alt-o' if is_storage_ng;
```

Comment:
synchronisation before the next key press?

Reply:
Problem is that this code is used in two other test modules, namely partitioning_warnings and partitioning_full_lvm. I've added wait_still_screen not to break test by introducing new needles, but I agree that this would be better.